test/cyclonus/install-cyclonus-exclude-sctp.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             - --pod-creation-timeout-seconds=30
             - --job-timeout-seconds=2
             - --server-protocol=TCP,UDP
-            - --exclude=sctp
+            - --exclude=sctp,end-port # TODO update kind cluster to use latest k8s version and remove endport exclusion
```

Comment:
this can be done by bumping this version here
https://github.com/Azure/azure-container-networking/blob/master/test/kind/kind.yaml#L7